app/models/runner.rb

```diff
@@ -19,13 +19,13 @@ def self.strategy_class
   end
 
   def self.management_active?
-    @management_active ||= begin
-      runner_management = CodeOcean::Config.new(:code_ocean).read[:runner_management]
-      if runner_management
-        runner_management[:enabled]
-      else
-        false
-      end
+    return @management_active unless @management_active.nil?
```

Comment:
How about `defined?(@management_active)`? 🤔 Could be a bit more clear..

Reply:
As discussed in person, I don't have a strong opinion here.

Reply:
Thanks for your input. As discussed, I went with `defined?` to cover `nil` values as well and have the respective code part easier to read (with an `if` rather than an `unless`).